vector/src/main/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewViewEvent.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.settings.devices.v2.overview
+
+import im.vector.app.core.platform.VectorViewEvents
+import org.matrix.android.sdk.api.session.Session
+
+sealed class SessionOverviewViewEvent : VectorViewEvents {
+    data class SelfVerification(val session: Session) : SessionOverviewViewEvent()
```

Comment:
Please remove `session`, this is not used.

Reply:
Fixed in [87c2b94](https://github.com/vector-im/element-android/pull/7139/commits/87c2b94ff91ac4f8765fe908b5a0924c595334ba)